### ISSUE TITLE
Abnormality of the musculoskeletal system

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5902]
**Preferred term label:** Abnormality of the musculoskeletal system

**Synonyms**

**Definition (free text, please give PubMed ID)** An abnormality of the muscular and/or skeletal system.

**Parent term (use hpo.jax.org/app)** Phenotypic abnormality HP:0000118

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be the parent term for: Abnormality of the skeletal system HP:0000924; Abnormality of the musculature HP:0003011

**Comment by pnrobinson** [comment]
this is https://hpo.jax.org/app/browse/term/HP:0033127
